source/errors.rst

```diff
@@ -186,7 +186,7 @@ contain a slug that already exists.
 
   {
     "status": 409,
-    "error": "Slug already exists",
+    "error": "Slug Already Exists",
```

Comment:
https://github.com/osuosl/timesync-node/blob/develop/src/errors.js#L169 shows `'The slug provided already exists'` as the correct form.